src/components/Filter.js

```diff
@@ -5,7 +5,8 @@ class Filter extends Component {
   static propTypes = {
     setFilter: PropTypes.func,
     style: PropTypes.object,
-    className: PropTypes.string
+    className: PropTypes.string,
+    text: PropTypes.string
```

Comment:
Would it make sense to rename this prop as `placeholder`, to align with the `textProperties` key and to make explicit how the value will be used?

Bonus points if you add a trailing comma.